ext/date/date_core.c

```diff
@@ -5640,12 +5645,17 @@ d_lite_leap_p(VALUE self)
 
 /*
  * call-seq:
- *    d.start  ->  float
+ *   start -> float or Infinity or -Infinity
```

Comment:
I think float implies it could be infinity, so I don't think we need to explicitly mention it.

Reply:
Fixed.